curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/66507c5f5766304f531cdfea.md

```diff
@@ -0,0 +1,90 @@
+---
+id: 66507c5f5766304f531cdfea
+title: Step 50
+challengeType: 1
+dashedName: step-50
+---
+
+# --description--
+
+Learning about functions for the first time can sometimes be a lot to take in. Before you move onto the next lessons, you will review what you have learned so far about about functions.
+
+In this step, you will need to do the following:
+
+- Create a function called `addTwoNumbers` that takes two parameters called `num1` and `num2`.
+- The function should return the sum of the two numbers.
+- Call the `addTwoNumbers` function and pass in the numbers `5` and `10`.
+- Then on a new line, call the `addTwoNumbers` function again and pass in the numbers `10` and `20`.
+
+
```

Comment:
> Let's make them think about this a bit more, since it's a review step.

Do you mean the result variable as the sum? Keeping result as `result` is making me think it to be a separate variable.

Reply:
Yes - I edited the suggestion. Good catch!